cicecore/cicedyn/dynamics/ice_transport_remap.F90

```diff
@@ -1508,7 +1540,160 @@ subroutine limited_gradient (nx_block, ny_block,   &
 
       enddo                     ! ij
 
-      end subroutine limited_gradient
+      end subroutine limited_gradient_cn_dbl
+
+!=======================================================================
+! Part 2 of the interface that allow cnx and cny to either matrices or scalars
```

Comment:
Where is Part 1?
Should this comment say "Part 2 of the interface that allows cnx and cny to be either..." ?

Reply:
Part 1  is the subroutine limited_gradient_cn_dbl, which I think should be renamed to limited_gradient_cn_2d as it is more correct.
Yes the "s" and "be" is missing. I will push an update with these and at the same time remove comments from ice_transport_remap